src/device-registry/utils/create-airqloud.js

```diff
@@ -511,6 +512,11 @@ const createAirqloud = {
               if (isSiteInAirQloud === true) {
                 site_ids.push(site._id);
               } else if (isSiteInAirQloud === false) {
+                logger.info(
+                  `Site ${site._id} does not belong to this AirQloud`
```

Comment:
the main part of the PR where we log values for the rest of the alternative flows.